### PR TITLE
update to support rails 7

### PR DESCRIPTION
**Pull request by cloudsbird**
Rails 7.0.0 is already launch, I update the gemspec and active record dependencies in gemfiles. Hope it helps, 

tested in my local environment

gemfile: 
`gem 'acts_as_paranoid',               '~> 0.7.3', git: 'https://github.com/cloudsbird/acts_as_paranoid', branch: 'rails_7_support'`

ruby: 3.0.0
rails: 7.0.0

### CODE REVIEW

File: acts_as_paranoid.gemspec [hunk 1]
```diff
@@ -20,8 +20,8 @@ Gem::Specification.new do |spec|
   spec.files = File.read("Manifest.txt").split
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activerecord", ">= 5.2", "< 7.0"
-  spec.add_runtime_dependency "activesupport", ">= 5.2", "< 7.0"
+  spec.add_runtime_dependency "activerecord", ">= 5.2", "<= 7.0"
+  spec.add_runtime_dependency "activesupport", ">= 5.2", "<= 7.0"
```
Comment (mvz) [inline acts_as_paranoid.gemspec:24]: Please use "< 7.1" instead, so patch updates to Rails will be allowed.

**Review by mvz — COMMENTED** [review]
Thanks, @cloudsbird! Just one small nit, see my comment. Also, can you please add an entry to the actions workflow?

File: acts_as_paranoid.gemspec (hunk 1 above)
Comment (cloudsbird) [reply acts_as_paranoid.gemspec:24]: okay sure

**Comment by cloudsbird** [comment]
Done @mvz

**Comment by cloudsbird** [comment]
I notice why error.. caused by ruby version I think

**Comment by cloudsbird** [comment]
@mvz Sorry can run workflow once more hehe..

**Comment by cloudsbird** [comment]
@mvz okay after researching about jruby, and reason why it is failed it is showed in here https://github.com/jruby/jruby/issues/6878 So I also will exclude jruby-9.3 from test, and wait till jruby-9.4 launched

**Comment by mvz** [comment]
Thanks, @cloudsbird!

**Pull request merged**